src/components/flyout/Flyout.js

```diff
@@ -408,6 +408,7 @@ class Flyout extends React.Component<Props, State> {
         }
 
         const overlayProps = {
+            className: 'flyout-overlay',
```

Comment:
wouldn't styles of the previously defined `flyout-overlay` class now apply to this overlay?

Reply:
maybe this should be `overlay-content` / `flyout-overlay-content` since the class is applied to the overlayContent component?